app/javascript/publishers/home.js

```diff
@@ -15,6 +15,20 @@ const SELECT_CURRENCY = '---';
 const BASIC_ATTENTION_TOKEN = 'BAT'
 const UNAVAILABLE = 'unavailable';
 
+function formatConvertedBalance(amount, currency) {
+  if (isNaN(amount) || amount === null ) {
+    return `${currency} ${UNAVAILABLE}`;
+  } else {
+    let amount = formatAmount(amount)
+    return `~ ${amount} ${currency}`;
```

Comment:
I think we need to change line 22 and 23 to:

```
let amountFormatted = formatAmount(amount)
return `~ ${amountFormatted} ${currency}`;
```
to avoid variable scoping problems.  Doing so fixed an issue I was seeing in which a converted overall currency was `NaN` after this js was run when changing default currency.

Reply:
Fixed